### ISSUE TITLE
Don't know the JSON argument array format to perform mobile: clickGesture

### ISSUE THREAD

**Issue by SHI-ZP** [appium/appium-inspector#547]
## Appium or Appium Desktop?

You are reporting an issue at the Appium Desktop repository. Appium Desktop is a wrapper around Appium. If you are having trouble running tests, it is much more likely that the problem you are encountering is not a problem with Appium Desktop but with Appium. For that reason we require you to have tried your testcase with Appium CLI before reporting issues. Check the checkbox below to confirm that you have proven your issue does _not_ reproduce on Appium itself:

- [x] I have verified that my issue does not occur with Appium and should be investigated as an Appium Desktop issue

## The problem

I do not know what the correct JSON argument array to perform mobile: clickGesture is:
<img width="533" alt="image" src="https://user-images.githubusercontent.com/16377386/182350859-b7f2673c-9547-4db3-99b7-02e8b032d18c.png">
I've tried [100, 100] as the JSON array. but does not work:
<img width="408" alt="image" src="https://user-images.githubusercontent.com/16377386/182351445-9aa9e5da-e596-410c-a1b5-33307daa8742.png">

Thanks for the help.

## Environment

- I am running Appium Desktop version:
<img width="260" alt="image" src="https://user-images.githubusercontent.com/16377386/182350992-ed95d9ec-d7bb-44cd-8dfe-5b2008b669e9.png">

- I am on (pick one):
    - [X] Mac
    - [ ] Windows
    - [ ] Linux

**Comment by jlipps** [comment]
According to the docs it would be something like one of the following, depending on what you are trying to do:

```js
[{"elementId":  "<id of element here>"}]
[{"elementId":  "<id of element here>", "x": <x-offset>, "y": <y-offset>}]
[{"x": <x-coord>, "y": <y-coord>}]
```

**Comment by SHI-ZP** [comment]
Thank you so much, @jlipps.

`[{"x": 100, "y": 100}]` works for me.

I found the appium 2.0 mobile: clickGesture from [here](https://appium.io/docs/en/writing-running-appium/android/android-mobile-gestures/), which does not mention the format for the Inspector use. Could you please share with me the docs you were referring to?

**Comment by jlipps** [comment]
the documentation for executeScript explains that the arguments for the function include a string script (in our case `mobile: clickGesture`) followed by an array of arguments. the array of arguments is what you see here. typically, by appium convention, `mobile:` commands have a single argument which is a JS object so we can have key parameter names and values.

In other words, I took the example from the docs (something like `driver.execute_script('mobile: longClickGesture', {'x': 100, 'y': 100, 'duration': 1000})`) and converted it to the underlying webdriver representation.
